guest-libs/verify_stark/README.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+A guest library to verify OpenVM stark proofs. The library could be used for Rollup state transition functions or other purposes.
+
+# Details 
+## Function definition Macro
+Macro `define_verify_stark_proof` can define a function to verify a stark proof. Users need to specify:
+- The function name.
+- An ASM file, which contains instructions to verify proofs of **a specific aggregation config**. The ASM file could be generated during the aggregation keygen.
```

Comment:
not quite clear here what verify proofs of a specific aggregation config means - do you mean verify proofs of a static VM config? (it's not the aggregation config that matters, but the app VM config)